src/components/VmDisks/style.css

```diff
@@ -27,3 +28,14 @@
     font-size: small;
     cursor: pointer;
 }
+
+.vmdisk-item {
```

Comment:
I would make these more generic, not just for `vmdisk`.
Isn't there same issue for networks?
What about potential future list of other objects on the same place?

@jniederm

Reply:
Provided we have time for this, it would surely be nice to have a component. I wouldn't go for anything complex like `AddRemoveListing` but rather use something purely visual like `AlignedListing`